src/views/WorkspaceDetail.vue

```diff
@@ -258,7 +258,7 @@ export default Vue.extend({
 
       if (this.localWorkspace !== null) {
         try {
-          const { data } = await api.renameWorkspace(this.workspace, this.localWorkspace);
+          const data = (await api.renameWorkspace(this.workspace, this.localWorkspace)).name;
```

Comment:
The following might be more readable (if it works correctly):

```suggestion
          const { name } = (await api.renameWorkspace(this.workspace, this.localWorkspace));
```

In order to use this, you would need to also change `data` to `name` in the following lines (so, rather than accepting this code suggestion, I would incorporate it into your own local edits and make sure it works as intended).

This is better because it reflects the actual usage of values in the variable names.

Reply:
That works!

See d4ba127